### DISCUSSION TITLE
Why not use @inlinable on functions?

### DISCUSSION THREAD

**Discussion by maximkrouk** [pointfreeco/swift-prelude#167] [Q&A] [answered]
Are there any pros to not using the @inlinalbe attribute?

**Comment by stephencelis** [comment, marked as answer]
I'm sure a lot of the generic functions in Prelude would benefit from inlining. Prelude's an experimental library, though, so we haven't focussed on performance in the slightest, especially when compared with our other libraries.

If you experiment with inlining on your fork and have any interesting findings to share, we'd love to hear them!
